app/experimenter/nimbus-ui/src/react-app-env.d.ts

```diff
@@ -6,3 +6,7 @@
 
 type DateTime = string;
 type ObjectField = Record<string, any> | string;
+
+type OutcomeSlugs = (string | null)[] | null;
+type Outcome = getConfig_nimbusConfig_outcomes | null | undefined;
```

Comment:
I noticed that this type is effectively `any[]` because we need an import in this module:

```javascript
import { getConfig_nimbusConfig_outcomes } from "./types/getConfig"
```

Not sure why typescript isn't complaining about that.